### PR TITLE
Implement Search for Survivors

### PR DESCRIPTION
**Pull request by jmharmon**

### CODE REVIEW

File: Mage/src/main/java/mage/abilities/effects/common/ShuffleGraveyardSourceEffect.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @Override
+    public ShuffleGraveyardSourceEffect copy() {
+        return new ShuffleGraveyardSourceEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player player = game.getPlayer(source.getControllerId());
+        if (player != null) {
+            player.shuffleGraveyard(source, game);
```
Comment (JRHerlehy) [inline Mage/src/main/java/mage/abilities/effects/common/ShuffleGraveyardSourceEffect.java]: Need to add the method definition for shuffleGraveyard to the Player interface and them implement the method in the PlayerImpl class. As of now, this method does not exist.

**Comment by JayDi85** [comment]
@jmharmon create new branch and pull request for premodern. That's PR for Search for Survivors only.

**Comment by jmharmon** [comment]
I'm having problem updating my copy of xmage code, can someone just close this request?

**Comment by JayDi85** [comment]
@jmharmon if you can't create new one pull request then I test and add your code manually later.

**Pull request closed without merging**